_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```

Comment:
This was added to demo github pages at https://macterra.github.io/Ulex/versions/1.1/ for #19